packages/docs-site/docs/ref/domain/syntax.mdx

```diff
@@ -5,4 +5,142 @@ pagination_next: null
+
+## Type Declarations
+
+A _type_ is a class of objects that a diagram works with. In the set-theory example, _Set_ is a type. The syntax for a type declaration is as follows:
+
+```
+type typename
+```
+
+This statement declares a type with name `typename`. After declaration, a type can be referred to in other parts of the _domain_ schema, as well as the _style_ and _substance_ schemas.
+
+For example, statement `type Set` declares a type with name `Set`. We can then use the name `Set` to later refer to this type.
+
+### Subtypes
+
+Conceptually, a type might be a _subtype_ of another type. As an example, in a chemistry domain, types _Hydrogen_ and _Oxygen_ might both be subtypes of the type _Atom_. _Penrose_ allows us to represent subtyping relations as follows:
+
+```
+type subtype_name <: supertype_name
+```
+
+Take the example of types _Hydrogen_, _Oxygen_, and _Atom_:
+
+```
+type Atom
+type Hydrogen <: Atom
+type Oxygen <: Atom
+```
+
+Subtyping describes "is-a" relationships between types: a Hydrogen "is-a[n]" Atom; an Oxygen "is-a[n]" Atom. Using the "is-a" interpretation, since an Atom "is-a[n]" Atom, inherently, for all type `A`, we have that `A` is a subtype of `A`. This interpretation of subtyping is consistent with "inheritance" in many object-oriented programming languages.
+
+_Penrose_ adopts this interpretation: if _Penrose_ expects to see an object with type `A`, then it will also accept an object of type `B`, as long as `B` is a subtype of `A`. We say that "`B` **matches** `A`."
+
+## Predicate Declarations
+
+A _predicate_ represents mathematical or logical statements regarding zero or more objects or other statements. To declare a predicate, we write
+
+```
+predicate predicate_name (argument_list)
+```
+
+where
+
+- `predicate_name` is the name of the predicate, which can be referred to in the _substance_ and _style_ schemas; and
+- `argument_list` describes the types of objects that this predicate works with. We will go more into its details below.
+
+The arguments in `argument_list` contains the types of objects that this predicate accepts, separated by commas. Here is a _domain_ schema that contains a couple of type declarations and four different, valid, predicate declarations:
+
+```
+type FirstType, SecondType
+
+predicate P1 (FirstType, SecondType)
+predicate P2 (SecondType a1, SecondType a2)
+predicate P3 ()
+predicate P4 (FirstType, SecondType, SecondType)
+```
+
+Notice how some declarations have names associated with the argument types (e.g. `P2` with argument names `a1` and `a2`), and some don't. These names to argument types are allowd and encouraged to enhance readability, but are not required.
+
+### The `Prop` Type
+
+A special type, _Prop_, is built into _Penrose_, and is available to be used as predicate arguments. The _Prop_ type represents predicates themselves. For example, the predicate _Not_ is declared in the set-theory example:
```

Comment:
Do we want to italicize _Penrose_ everywhere it appears?

Reply:
I am fine with both, but I do want to be consistent

Reply:
@joshsunshine Any ideas about whether "Penrose" should be italicized?

Reply:
We smallcap Penrose in academic documents, but I don't think it's really necessary for the language reference.

Reply:
I will not italicize any occurence of "Penrose"